.buildkite/create_integration_testing_worksheet.py

```diff
@@ -6,61 +6,106 @@
 # Environment Variable/s:
     * GOOGLE_APPLICATION_CREDENTIALS = Your service account key.
 """
-import itertools
 import os
 
 import gspread
 from oauth2client.service_account import ServiceAccountCredentials
 
```

Comment:
Nitpick, for "beautiful" codes to read: 😸 
1. Order your `import` statements alphabetically.
2. Put a new line after the last `import` and before the first `from ... import ...`.